src/sentry/issues/endpoints/group_events.py

```diff
@@ -38,29 +50,60 @@ class GroupEventsError(Exception):
     pass
 
 
+@extend_schema(tags=["Events"])
 @region_silo_endpoint
 class GroupEventsEndpoint(GroupEndpoint, EnvironmentMixin):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
     }
     owner = ApiOwner.ISSUES
 
+    @extend_schema(
+        operation_id="List an Issue's Events",
+        parameters=[
+            GlobalParams.ORG_ID_OR_SLUG,
+            IssueParams.ISSUES_OR_GROUPS,
+            IssueParams.ISSUE_ID,
+            GlobalParams.START,
+            GlobalParams.END,
+            GlobalParams.STATS_PERIOD,
+            GlobalParams.ENVIRONMENT,
+            OpenApiParameter(
+                name="full",
+                type=OpenApiTypes.BOOL,
+                location=OpenApiParameter.QUERY,
+                description="If this is set to true, the event payload will include the full event body, including the stacktrace. Set to true to enable.",
+                required=False,
+            ),
+            OpenApiParameter(
+                name="sample",
+                type=OpenApiTypes.BOOL,
+                location=OpenApiParameter.QUERY,
+                description="Return events in pseudo-random order. This is deterministic so an identical query will always return the same events in the same order.",
+                required=False,
+            ),
+            OpenApiParameter(
+                name="query",
+                location=OpenApiParameter.QUERY,
+                type=OpenApiTypes.STR,
+                description="An optional search query for filtering events.",
+                required=False,
+            ),
+        ],
+        responses={
+            200: inline_sentry_response_serializer(
+                "GroupEventsResponseDict", list[SimpleEventSerializerResponse]
+            ),
+            400: RESPONSE_BAD_REQUEST,
+            401: RESPONSE_UNAUTHORIZED,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=EventExamples.GROUP_EVENTS_SIMPLE,
+    )
     def get(self, request: Request, group: Group) -> Response:
         """
-        List an Issue's Events
-        ``````````````````````
-
         This endpoint lists an issue's events.
```

Comment:
nit: to mimic more recent docs, maybe use "Return a list of error events bound to an issue" from the OpenApiExample